plugin/storage/es/esRollover.py

```diff
@@ -112,8 +112,10 @@ def create_index(client, name):
     try:
         create.do_action()
     except curator.exceptions.FailedExecution as e:
-        if "index_already_exists_exception" not in str(e) and "resource_already_exists_exception" not in str(e):
+        if ("index_already_exists_exception" or "resource_already_exists_exception") in str(e):
```

Comment:
Doesn't OR of two strings simply return the first one?

Reply:
ohh this would not work. My python skills... 

Actually `str(e)` never returns this string. There is a parameter `ignore_existing=True` I would use that instead. If the index exists it logs it.

Index jaeger-span-000001 already exists.